Build fails on fresh n8n-nodes-starter clone

Cloning a fresh copy of the `n8n-nodes-starter` repository and running the development server results in a build failure with multiple TypeScript errors related to `NodeConnectionType`.

### **Command Run**

```bash
npm run dev
```

### **Log Results**

```text
[build] nodes/Example/Example.node.ts(7,10): error TS6133: 'NodeConnectionType' is declared but its value is never read.
[build] nodes/Example/Example.node.ts(20,12): error TS2693: 'NodeConnectionType' only refers to a type, but is being used as a value here.
[build] nodes/Example/Example.node.ts(21,13): error TS2693: 'NodeConnectionType' only refers to a type, but is being used as a value here.
[build] nodes/GithubIssues/GithubIssues.node.ts(1,10): error TS6133: 'NodeConnectionType' is declared but its value is never read.
[build] nodes/GithubIssues/GithubIssues.node.ts(21,12): error TS2693: 'NodeConnectionType' only refers to a type, but is being used as a value here.
[build] nodes/GithubIssues/GithubIssues.node.ts(22,13): error TS2693: 'NodeConnectionType' only refers to a type, but is being used as a value here.
[build] 4:45:36 PM - Found 6 errors. Watching for file changes.
n8n exited with code 1
```

--- a/nodes/Example/Example.node.ts
+++ b/nodes/Example/Example.node.ts
@@ -4,7 +4,7 @@ import type {
 	INodeType,
 	INodeTypeDescription,
 } from 'n8n-workflow';
-import { NodeConnectionType, NodeOperationError } from 'n8n-workflow';
+import { NodeConnectionTypes, NodeOperationError } from 'n8n-workflow';
 
 export class Example implements INodeType {
 	description: INodeTypeDescription = {
@@ -17,8 +17,8 @@ export class Example implements INodeType {
 		defaults: {
 			name: 'Example',
 		},
-		inputs: [NodeConnectionType.Main],
-		outputs: [NodeConnectionType.Main],
+		inputs: [NodeConnectionTypes.Main],
+		outputs: [NodeConnectionTypes.Main],
 		usableAsTool: true,
 		properties: [
 			// Node properties which the user gets displayed and

--- a/nodes/GithubIssues/GithubIssues.node.ts
+++ b/nodes/GithubIssues/GithubIssues.node.ts
@@ -1,4 +1,4 @@
-import { NodeConnectionType, type INodeType, type INodeTypeDescription } from 'n8n-workflow';
+import { NodeConnectionTypes, type INodeType, type INodeTypeDescription } from 'n8n-workflow';
 import { issueDescription } from './resources/issue';
 import { issueCommentDescription } from './resources/issueComment';
 import { getRepositories } from './listSearch/getRepositories';
@@ -18,8 +18,8 @@ export class GithubIssues implements INodeType {
 			name: 'GitHub Issues',
 		},
 		usableAsTool: true,
-		inputs: [NodeConnectionType.Main],
-		outputs: [NodeConnectionType.Main],
+		inputs: [NodeConnectionTypes.Main],
+		outputs: [NodeConnectionTypes.Main],
 		credentials: [
 			{
 				name: 'githubIssuesApi',

--- a/package.json
+++ b/package.json
@@ -8,8 +8,8 @@
 		"n8n-community-node-package"
 	],
 	"author": {
-		"name": "Elias Meire",
-		"email": "elias@meire.dev"
+		"name": "",
+		"email": ""
 	},
 	"repository": {
 		"type": "git",

